### ISSUE TITLE
GARDEN pages for CHS/Lookit Website Merger

### ISSUE THREAD

**Issue by ianchandlercampbell** [lookit/lookit-api#1156]
**TL;DR**: Children Helping Science/Lookit merger web plan; childrenhelpingscience.com should be displayed as parent site, host can be lookit.mit.edu; Project GARDEN will need a subsite childrenhelpingscience.com/project-garden with multiple pages

**Narrative**

Childrenhelpingscience.com and lookit will be merging late spring/early summer to be a "one-stop shop" for families who wish to participate in research online, building on the aspiration of the current Lookit home page ("Children Helping Science: powered by Lookit)

I've included an initial idea for a website map that can serve as the basis for the new childrenhelpingscience.com, including the home page, Project GARDEN subsite and pages, and how this can all integrate into the existing lookit.mit.edu site. There is an editable file to use at https://app.diagrams.net/ as well as a picture of the current output. 

Looking forward to further discussions as we move toward the new Children Helping Science site!

**Acceptance Criteria**
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] TBD
- [ ] Scoping
- [ ] Testing

![CHS-Lookit_Website map drawio_v2](https://user-images.githubusercontent.com/75502403/228053833-19e7c486-84f6-41c4-9f84-710df8a61916.svg)
sign-off

**Implementation Notes**
Please see the suggested diagram files for current suggestions!

**Comment by ianchandlercampbell** [comment]
There are a few minor differences, but they're essentially the same-- I've edited so only the editable .svg file is there now, as GitHub previews this properly. Feel free to use this as a base for editing y'alls plan as we move forward!

I can make another diagram for the suggested family experience as a supplement to this sitemap later this week, thank you for thinking through these options!
